lib/active_fedora/orders/list_node.rb

```diff
@@ -59,6 +59,19 @@ def target
         end
     end
 
+    def target_id
```

Comment:
I can almost see `target_id` and `proxy_in_id` collapse into a few shorter methods, but maybe it's best to leave that optimization for later?